dev/aarch64_opt/src/poly_reduce_asm_opt.S

```diff
@@ -9,7 +9,7 @@
 /* simpasm: header-end */
 
 /* Barrett reduction */
-.macro barrett_reduce a
+.macro mlk_barrett_reduce a
```

Comment:
same here.

Reply:
The script shouldn't have renamed this -- will manually undo.